src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -265,9 +265,11 @@
              let-row="data.row"
              let-value="data.value">
   <div class="position-relative">
```

Comment:
We can remove this bootstrap css and check if thats what causing issue ?
Even if not, we can get rid of this.